src/com/lftechnology/java/training/srijan/basics/formatter/LoggerFormatter.java

```diff
@@ -0,0 +1,33 @@
+package com.lftechnology.java.training.srijan.basics.formatter;
+
+import java.io.IOException;
+import java.util.logging.Formatter;
+import java.util.logging.FileHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+public class LoggerFormatter {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)